### ISSUE TITLE
GNOME - Verify removal of our MessageDialog class

### ISSUE THREAD

**Issue by nlogozzo** [NickvisionApps/Application#76]
gir.core v0.4.0 should've brought `Adw.MessageDialog` with it, allowing us to remove our wrapper class.

- [x] Application (https://github.com/NickvisionApps/Application/commit/98cd11e7ddd5f179766c4c67a4c6f665986f7bea)
- [x] Cavalier (https://github.com/NickvisionApps/Cavalier/pull/89)
- [x] Denaro (https://github.com/NickvisionApps/Denaro/pull/619)
- [x] Parabolic (https://github.com/NickvisionApps/Parabolic/pull/527)
- [x] Tagger (https://github.com/NickvisionApps/Tagger/pull/228)

**Comment by fsobolev** [comment]
`Adw.MessageDialog` was available before 0.4.0, we even used it inside our wrapper

```
public partial class MessageDialog
{
    private readonly Adw.MessageDialog _dialog;
```

The reason for wrapper as I understand as it allowed us for easier dialogs creations because some required properties are set in construtor.

**Comment by nlogozzo** [comment]
Ah well either way for the once/twice it's used in each app..Denaro I think 4/5 I think it's worth removing and using gir.core's version...
